### PR TITLE
@eessex => Include Email Signup and Read More from Reaction

### PR DESCRIPTION
**Pull request by kanaabe**
email-and-truncation

- Checks if user is signed up for editorial emails 
- Adds a signup route that sends a welcome email

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by kanaabe** [comment]
@eessex bump

File: desktop/apps/article2/__tests__/routes.test.js [hunk 1]
```diff
@@ -92,94 +93,160 @@ describe('Article Routes', () => {
...
+        )
+        Article.prototype.fetchWithRelated = sinon.stub().yieldsTo('success', data)
+        RoutesRewireApi.__Rewire__('Article', Article)
+        amp(req, res, next)
+        next.callCount.should.equal(1)
+        done()
+      })
+
+      it('redirects to the correct slug if it does not match', (done) => {
+        const data = {
```
Comment (eessex) [inline desktop/apps/article2/__tests__/routes.test.js]: Find this test very ambiguous -- what thing is not matching?

**Comment by eessex** [comment]
Just one comment about vague test language -- not necessary to address in this PR -- feel free to merge if ready.

**Pull request merged**